static/script/sketch.js

```diff
@@ -0,0 +1,356 @@
+// this will load from localStorage if it exists, otherwise it will load 0 as the default value
+const savedScore = localStorage.getItem('userRecord') || 0;
+
+// this is a giant list of all the variables used in the sketch, let's make this smaller
+let trex,
+    flyngDino,
+    ground,
+    clouds,
+    moon,
+
+    texTrex,
+    trexSprint,
+    trexCollide,
+    texFlyingDino,
+
+    // clouds and obstacles
+    cloudsGroup,
+    obstacleGroup,
+    flyingDinoGroup,
+    objObstacle1,
+
+    // death screen
+    gameOverTxt,
+    restart,
+
+    // info
+    score = 0,
+    maximumScore = savedScore,
+    endGame = false,
+    bg = 156,
+    moonOpacity = 0,
+    day = true,
+    canJump = true,
+    onGround = true,
+    isKeyUpPressed = false,
+    isKeyDownPressed = false,
+
+
+    // sounds
+    jumpingSound,
+    collideSound;
+
+function preload() {
+  // Pterodactylus dinossaur enemy flying animation
+  texFlyingDino = loadAnimation("/assets/bird1.png", "/assets/bird2.webp");
+
+  // Trex animations
+  texTrex = loadAnimation("/assets/trex1.png", "/assets/trex2.png", "/assets/trex3.webp", "/assets/trex4.webp");
+  trexSprint = loadAnimation("/assets/TrexDown1.png", "/assets/TrexDown2.png");
+  trexCollide = loadImage("/assets/collide.webp");
+
+  // Player sound effects
+  jumpingSound = loadSound("/sounds/collided.wav");
+  collideSound = loadSound("/sounds/jump.wav");
+}
+
+function setup() {
+  /*let canvas = document.createElement('canvas');
+  canvas.width = windowWidth;
+  canvas.height = windowHeight;
+
+  let ctx = canvas.getContext('2d');
+
+  ctx.fillStyle = '#000000';
+  ctx.fillRect(0, 0, windowWidth, windowHeight);*/

```

Comment:
ignore this, I was trying to create it without the library and see how it would interact with it